doc/reference/security.rst

```diff
@@ -93,8 +91,233 @@ access to the field:
 
 .. todo::
 
-    field access groups apply to administrator in fields_get but not in
+    field access groups apply to the Superuser in fields_get but not in
     read/write...
 
-.. _foo: http://google.com
-.. _time module: https://docs.python.org/2/library/time.html
+.. _time module: https://docs.python.org/3/library/time.html
+
+
+.. _reference/security/pitfalls:
+
+Security Pitfalls
+=================
+
+As a developer, it is important to understand the security mechanisms and avoid
+common mistakes leading to unsecure code.
+
+Unsafe Public Methods
+---------------------
+
+Any public method can be execute via a :ref:`RPC call
+<webservices/odoo/calling_methods>` with the chosen parameters. The methods
+starting with a ``_`` are considered as private in Odoo and not callable from
```

Comment:
```suggestion
starting with a ``_`` are not callable from
```
they're not private in the sense readers would usually understand: they're often very much intended to be used or overridden, just not callable over RPC.